frontend/packages/dev-console/src/components/topology/components/groups/HelmRelease.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import {
+  WithSelectionProps,
+  Layer,
+  useHover,
+  Node,
+  createSvgIdUrl,
+  useDragNode,
+  observer,
+  useCombineRefs,
+} from '@console/topology';
+import NodeShadows, { NODE_SHADOW_FILTER_ID_HOVER, NODE_SHADOW_FILTER_ID } from '../NodeShadows';
+import SvgBoxedText from '../../../svg/SvgBoxedText';
+import './HelmRelease.scss';
+
+export type HelmReleaseProps = {
+  element: Node;
+  dragging: boolean;
+} & WithSelectionProps;
+
+const HelmRelease: React.FC<HelmReleaseProps> = ({ element, dragging }) => {
+  const [hover, hoverRef] = useHover();
+  const { x, y, width, height } = element.getBounds();
+  const dragNodeRef = useDragNode()[1];
+  const dragLabelRef = useDragNode()[1];
+  const refs = useCombineRefs(dragNodeRef, hoverRef);
+  return (
+    <g>
+      <NodeShadows />
+      <Layer id={dragging ? undefined : 'groups'}>
+        <rect
+          ref={refs}
+          className="odc-helm-release"
+          x={x}
+          y={y}
+          width={width}
+          height={height}
+          rx="5"
+          ry="5"
+          filter={createSvgIdUrl(
+            hover || dragging ? NODE_SHADOW_FILTER_ID_HOVER : NODE_SHADOW_FILTER_ID,
+          )}
+        />
+      </Layer>
+      {element.getLabel() && (
+        <>
+          <SvgBoxedText
+            className="odc-helm-release__label odc-base-node__label"
+            x={x + width / 2}
+            y={y + height + 20}
+            paddingX={8}
+            paddingY={4}
+            kind="HelmRelease"
```

Comment:
Instead of hard coding `HelmRelease` can we use `element.getData().kind`? Or is there a model for Helm release that we can use?

Reply:
There is no model or kind for this group.